packages/react-composites/src/composites/CallComposite/adapter/CallingSoundSubscriber.ts

```diff
@@ -45,6 +45,11 @@ export class CallingSoundSubscriber {
         this.soundsLoaded.callRingingSound.loop = false;
         this.soundsLoaded.callRingingSound.pause();
       }
+      if (this.call.state === 'Disconnecting') {
```

Comment:
how come this is a new if statement and not in the above if statement?

Reply:
instead of if connected || disconnecting || disconnected - to avoid it still playing if it gets into other non-ringing states, could this be `if state !== connecting && state !== connecting ringing`?

Reply:
yup I like that, will update